src/vm/clsload.cpp

```diff
@@ -118,15 +118,7 @@ PTR_Module ClassLoader::ComputeLoaderModuleWorker(
     {
         if (pDefinitionModule->IsCollectible())
             goto ComputeCollectibleLoaderModule;
-        if (!pDefinitionModule->GetAssembly()->IsDomainNeutral())
-        {
-            pFirstNonSharedLoaderModule = pDefinitionModule;
-        }
-        else
-        if (!pDefinitionModule->IsSystem())
-        {
-            pFirstNonSystemSharedModule = pDefinitionModule;
-        }
+        pFirstNonSharedLoaderModule = pDefinitionModule;
```

Comment:
And you can fold `pFirstNonSharedLoaderModule` and `pLoaderModule` into a single `pLoaderModule` local. They do not need to be two different variables anymore.